### PR TITLE
add easyconfig with OpenFOAM 2.2.2 + required patches

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review (test report is coming up)

File: easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.0_esmumps-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = 'SCOTCH'
+version = '6.0.0_esmumps'
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.0_esmumps-intel-2015a.eb]: split into `versionsuffix`?

File: easybuild/easyconfigs/o/OpenFOAM/cleanup-OpenFOAM-2.2.2.patch [hunk 2]
```diff
@@ -0,0 +1,235 @@
+diff -ru OpenFOAM-2.2.2.orig/etc/bashrc OpenFOAM-2.2.2/etc/bashrc
```
Comment (wpoely86) [inline easybuild/easyconfigs/o/OpenFOAM/cleanup-OpenFOAM-2.2.2.patch]: add a comment what this patch all does?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/46658b7d0ce1af261ae0 for a full test report.

File: easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.0_esmumps-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.0_esmumps-intel-2015a.eb]: no, it's actually a separate version, it's not a variant build, see http://gforge.inria.fr/frs/shownotes.php?release_id=7732

but apparently there's a 6.0.3 instead, so let me switch to that instead

File: easybuild/easyconfigs/o/OpenFOAM/cleanup-OpenFOAM-2.2.2.patch (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/o/OpenFOAM/cleanup-OpenFOAM-2.2.2.patch]: done

**Comment by boegel** [comment]
still works with SCOTCH v6.0.3 (can't submit a test report because of https://github.com/hpcugent/easybuild-framework/issues/1198, though)

merging this in, thanks @wpoely86 for the review!

**Pull request merged**